components/popup/overview.md

```diff
@@ -15,20 +15,32 @@ The <a href = "https://www.telerik.com/blazor-ui/popup" target="_blank">Blazor P
 ## Creating Blazor Popup
 
 1. Add the `<TelerikPopup>` tag to a Razor file.
-1. Obtain the component reference through `@ref`.
-1. Use the [Show](#popup-reference-and-methods) method to display the `<TelerikPopup>`.
+1. Set the `AnchorSelector` parameter to a CSS selector, which points to the HTML element that the Popup will align with.
+1. [Obtain the component reference to show and hide the Popover programmatically](#popup-reference-and-methods).
```

Comment:
When the entire sentence is a link, you can include the ending punctuation as part of the link. Otherwise, it looks rather awkward.